core/src/code/errors.rs

```diff
@@ -132,6 +132,20 @@ pub enum ImportError {
     /// The signature of an imported function is invalid.
     #[display(fmt = "Invalid function signature for imported function with index `{_0}`")]
     InvalidImportFnSignature(u32),
+    /// Unexpected import kind.
+    #[display(fmt = "Unexpected import kind `{kind}` with index `{index}`")]
+    UnexpectedImportKind {
+        /// Kind of the import.
+        kind: &'static str,
```

Comment:
`&'static str` is fat pointer, plz use `&'static &'static str` or enum

Reply:
It looks like &str is better option here

Fixed